code/modules/antagonists/cult/cult_structures.dm

```diff
@@ -32,6 +34,8 @@
 	. = ..()
 	. += "<span class='notice'>\The [src] is [anchored ? "":"not "]secured to the floor.</span>"
 	if((iscultist(user) || isobserver(user)) && cooldowntime > world.time)
+		if(cultist_examine_message)
+			. += "<span class='cult'>[cultist_examine_message]</span>"
 		. += "<span class='cult italic'>The magic in [src] is too weak, [p_they()] will be ready to use again in [DisplayTimeText(cooldowntime - world.time)].</span>"
```

Comment:
this should be under in an `if(cooldowntime > world.time)` check

Reply:
thanks